tests/lib/rules/boolean-prop-naming.js

```diff
@@ -91,222 +86,205 @@ ruleTester.run('boolean-prop-naming', rule, {
-    code: [
-      'class Hello extends React.Component {',
-      '  static propTypes = {isSomething: PropTypes.bool};',
-      '  render () { return <div />; }',
-      '}'
-    ].join('\n'),
+    code: `
+      class Hello extends React.Component {
+        static propTypes = {isSomething: PropTypes.bool};
+        render () { return <div />; }
+      }
+    `,
     options: [{
       rule: '^is[A-Z]([A-Za-z0-9]?)+'
     }],
     parser: 'babel-eslint'
   }, {
-    // ES6 components with static class properties and React.PropTypes
-    code: [
-      'class Hello extends React.Component {',
-      '  static propTypes = {isSomething: React.PropTypes.bool};',
-      '  render () { return <div />; }',
-      '}'
-    ].join('\n'),
+    code: `
+      class Hello extends React.Component {
+        static propTypes = {isSomething: React.PropTypes.bool};
+        render () { return <div />; }
+      }
+    `,
     options: [{
       rule: '^is[A-Z]([A-Za-z0-9]?)+'
     }],
     parser: 'babel-eslint'
   }, {
-    // ES6 components with static class properties an non-booleans
-    code: [
-      'class Hello extends React.Component {',
-      '  static propTypes = {something: PropTypes.any};',
-      '  render () { return <div />; }',
-      '}'
-    ].join('\n'),
+    code: `
+      class Hello extends React.Component {
+        static propTypes = {something: PropTypes.any};
+        render () { return <div />; }
+      }
+    `,
     options: [{
       rule: '^is[A-Z]([A-Za-z0-9]?)+'
     }],
     parser: 'babel-eslint'
   }, {
-    // ES6 components and Flowtype booleans
-    code: [
-      'class Hello extends React.Component {',
-      '  props: {isSomething: boolean};',
-      '  render () { return <div />; }',
-      '}'
-    ].join('\n'),
+    code: `
+      class Hello extends React.Component {
+        props: {isSomething: boolean};
+        render () { return <div />; }
+      }
+    `,
     options: [{
       rule: '^is[A-Z]([A-Za-z0-9]?)+'
     }],
     parser: 'babel-eslint'
   }, {
-    // ES6 components and Flowtype non-booleans
-    code: [
-      'class Hello extends React.Component {',
-      '  props: {something: any};',
-      '  render () { return <div />; }',
-      '}'
-    ].join('\n'),
+    code: `
+      class Hello extends React.Component {
+        props: {something: any};
+        render () { return <div />; }
+      }
+    `,
     options: [{
       rule: '^is[A-Z]([A-Za-z0-9]?)+'
     }],
     parser: 'babel-eslint'
   }, {
-    // Stateless components
-    code: [
-      'var Hello = ({isSomething}) => { return <div /> }',
-      'Hello.propTypes = {isSomething: PropTypes.bool};'
-    ].join('\n'),
+    code: `
+      var Hello = ({isSomething}) => { return <div /> }
+      Hello.propTypes = {isSomething: PropTypes.bool};
+    `,
     options: [{
       rule: '^is[A-Z]([A-Za-z0-9]?)+'
     }],
     parser: 'babel-eslint'
   }, {
-    // Functional components and Flowtype booleans
```

Comment:
looks like your codemod removed a bunch of comments. please restore them all

Reply:
Woops! My bad... I found the mistake 😄 I'll re-add all of them. I think most are actually in this rule where every test has a comment.